CharlieBackend.Api.UnitTes/DashboardServiceTest.cs

```diff
@@ -464,13 +464,6 @@ public async Task GetStudentGroupResult()
             Assert.Empty(requestWithoutGroupOnCourse.Data.AverageStudentGroupsMarks);
             Assert.Empty(requestWithoutGroupOnCourse.Data.AverageStudentGroupsVisits);
         }
-
-        protected override Mock<IUnitOfWork> GetUnitOfWorkMock()
-        {
-            var mock = new Mock<IUnitOfWork>();
-            return mock;
-        }
-
         private static ClaimsPrincipal GetClaimsPrincipal(UserRole role, long id, string name)
```

Comment:
Please, leave 1 blank line between methods after deleting this code.
And update everywhere else.